### PR TITLE
replaced File().exists with fileExists

### PR DESCRIPTION
**Pull request by de-code**
extension of #55

hopefully requires less permissions.

### CODE REVIEW

File: jenkinsfiles/Jenkinsfile.grobid [hunk 1]
```diff
@@ -18,7 +18,7 @@ elifePipeline {
         stage "Build Image", {
             dir('grobid') {
                 def dockerfile = "Dockerfile"
-                if (new File("Dockerfile.crf").exists()) {
+                if (fileExists("Dockerfile.crf")) {
```
Comment (NuclearRedeye) [inline jenkinsfiles/Jenkinsfile.grobid:21]: Initially thought looking at the docs that it's supposed to be written `fileExists path/to/file`, but seems from a quick google that `fileExists(path/to/file)` also works.

**Review by NuclearRedeye — APPROVED** [review]
I've no way to test this, from a quick google and read it seems to me this should work OK.

File: jenkinsfiles/Jenkinsfile.grobid (hunk 1 above)
Comment (de-code) [reply jenkinsfiles/Jenkinsfile.grobid:21]: I believe the `if` statement mandates brackets but I am not so sure about the Groovy syntax either.

This answer has both versions in it: https://stackoverflow.com/a/38539009/8676953

**Comment by de-code** [comment]
This seem to have worked well: https://alfred.elifesciences.org/job/process/job/process-grobid-docker/64/console
(it's also picking up the correct file: `Dockerfile.crf`)

**Pull request merged**